src/_tests/fixturedActions.test.ts

```diff
@@ -38,7 +38,7 @@ async function testFixture(dir: string) {
     response,
     (expr: string) => Promise.resolve(files[expr] as string),
     (name: string, _until?: Date) => name in downloads ? downloads[name] : 0,
-    readJSON(derivedPath).now
+    (readJSON(derivedPath) as NotFail | BotFail).now
```

Comment:
See below -- maybe `BotResult` instead of these two?

Reply:
Done :heavy_check_mark: